### DISCUSSION TITLE
How to reinstall OS w/ minimal impact on existing installation?

### DISCUSSION THREAD

**Discussion by FunkySalamander** [elementary/installer#728] [Q&A]
I have an existing Elementary OS Horus installation I'd like to reinstall but only to do kind of a refresh, as opposed to a nuke, of the existing installation.

I don't want to lose my home partition's configuration of things, eg. Email, Calendar, Browser, VLC, Top & Bottom Panels, etc.

I do want to lose non-default configurations like window decorations from other non standard DE's that had been installed alongside Pantheon.

And I do want to fix stuff like missing Desktop icon in System Settings.

**Comment by lenemter** [comment]
I don't think this is possible. All configuration files for Pantheon and other DEs are stored in your Home folder. So in order to reset everything, full reinstall is needed.
